board/app/models/user.php

```diff
@@ -1,52 +1,60 @@
 <?php
 
-class User extends AppModel {
-//The following constants are declared to avoid magic numbers
-    const MIN_USERNAME_LENGTH = 2;
-    const MIN_FIRST_NAME_LENGTH = 2;
-    const MIN_LAST_NAME_LENGTH = 2;
-    const MIN_EMAIL_LENGTH = 4;
-    const MIN_PASSWORD_LENGTH = 8;
-
-    const MAX_USERNAME_LENGTH = 20;
-    const MAX_FIRST_NAME_LENGTH = 30;
-    const MAX_LAST_NAME_LENGTH = 30;
-    const MAX_EMAIL_LENGTH = 30;
-    const MAX_PASSWORD_LENGTH = 20;
-
-    public $is_validated = true;
+class User extends AppModel 
+{
 
 /*
-*Registration Length Validation.
-*to make sure that a specific field is only limited 
-*   to a specific number of characters.
+* constants are used to avoid magic numbers
 */
+const MIN_USERNAME_LENGTH = 2;
+const MIN_FIRST_NAME_LENGTH = 2;
+const MIN_LAST_NAME_LENGTH = 2;
+const MIN_EMAIL_LENGTH = 4;
+const MIN_PASSWORD_LENGTH = 8;
+
+const MAX_USERNAME_LENGTH = 20;
+const MAX_FIRST_NAME_LENGTH = 30;
+const MAX_LAST_NAME_LENGTH = 30;
+const MAX_EMAIL_LENGTH = 30;
```

Comment:
Regarding MIN/MAX_EMAIL_LENGTH, what would happen if the user happens to have a legit but short/long email address? :sweat_smile: